<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>photo search</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Proxima Nova</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Найденные сходства</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="table_result">
        <property name="minimumSize">
         <size>
          <width>211</width>
          <height>211</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
border-radius: 10px;
background-color: rgb(35, 35, 35);</string>
        </property>
        <column>
         <property name="text">
          <string>id</string>
         </property>
         <property name="font">
          <font>
           <family>Proxima Nova</family>
          </font>
         </property>
         <property name="background">
          <color>
           <red>45</red>
           <green>45</green>
           <blue>45</blue>
          </color>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>45</red>
            <green>45</green>
            <blue>45</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>имя</string>
         </property>
         <property name="font">
          <font>
           <family>Proxima Nova</family>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ссылка на профиль</string>
         </property>
         <property name="font">
          <font>
           <family>Proxima Nova</family>
          </font>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>40</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>132</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Proxima Nova</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>сортировка по имени:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_name_sort">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid white;
border-radius: 3px;
color: white;
height: 20px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_sort">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border-radius: 9px;
	height: 22px;
}

QPushButton:hover {
	background-color: rgb(208, 208, 208);
	color: rgb(45, 45, 45);
}

QPushButton:pressed {
	background-color: rgb(207, 207, 207);
}</string>
          </property>
          <property name="text">
           <string>ок</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>40</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Proxima Nova</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>открыть сылки с</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="edit_spin_first">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid white;
border-radius: 3px;
color: white;
height: 20px;
</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Proxima Nova</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>по</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="edit_spin_second">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid white;
border-radius: 3px;
color: white;
height: 20px;
</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_open_url">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border-radius: 9px;
	height: 22px;
}

QPushButton:hover {
	background-color: rgb(208, 208, 208);
	color: rgb(45, 45, 45);
}

QPushButton:pressed {
	background-color: rgb(207, 207, 207);
}</string>
          </property>
          <property name="text">
           <string>ок</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="verticalSpacing">
       <number>40</number>
      </property>
      <item row="2" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="pic">
        <property name="minimumSize">
         <size>
          <width>211</width>
          <height>211</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px;
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Proxima Nova</family>
              <pointsize>10</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: white;
}

QLabel:hover {
	
}</string>
            </property>
            <property name="text">
             <string>пол:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btn_woman">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Proxima Nova</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton {
	color: white;
}

QRadioButton::indicator {
	width: 11px;
	height: 11px;
	border-radius: 5px;
}

QRadioButton::indicator::unchecked{ 
	border: 1px solid rgb(75, 75, 75);
	border-radius: 5px;
	background-color: rgb(75, 75, 75);
	width: 11px;
	height: 11px;
}

QRadioButton::indicator::checked{
	border: 3px solid;
	border-color: rgb(75, 75, 75);
	border-radius: 6px;
	background-color: white;
	width: 7px;
	height: 7px;
}</string>
            </property>
            <property name="text">
             <string>Ж</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btn_man">
            <property name="font">
             <font>
              <family>Proxima Nova</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton {
	color: white;
}

QRadioButton::indicator {
	width: 11px;
	height: 11px;
	border-radius: 5px;
}

QRadioButton::indicator::unchecked{ 
	border: 1px solid rgb(75, 75, 75);
	border-radius: 5px;
	background-color: rgb(75, 75, 75);
	width: 11px;
	height: 11px;
}

QRadioButton::indicator::checked{
	border: 3px solid;
	border-color: rgb(75, 75, 75);
	border-radius: 6px;
	background-color: white;
	width: 7px;
	height: 7px;
}</string>
            </property>
            <property name="text">
             <string>M</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_add">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Proxima Nova</family>
              <pointsize>10</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;доп проверка:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="btn_add_search">
            <property name="styleSheet">
             <string notr="true">QCheckBox {
	color: white;
}

QCheckBox::indicator {
	width: 11px;
	height: 11px;
	border-radius: 5px;
}

QCheckBox::indicator::unchecked{ 
	border: 1px solid rgb(75, 75, 75);
	border-radius: 5px;
	background-color: rgb(75, 75, 75);
	width: 11px;
	height: 11px;
}

QCheckBox::indicator::checked{
	border: 3px solid;
	border-color: rgb(75, 75, 75);
	border-radius: 6px;
	background-color: white;
	width: 7px;
	height: 7px;
}</string>
            </property>
            <property name="text">
             <string>да/нет</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_search_photo">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border-radius: 9px;
	height: 22px;
}

QPushButton:hover {
	background-color: rgb(208, 208, 208);
	color: rgb(45, 45, 45);
}

QPushButton:pressed {
	background-color: rgb(207, 207, 207);
}

QToolTip {
	
	background-color: rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>НАЙТИ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>500</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Rex Bold </family>
            <pointsize>18</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Выберите фото</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="btn_upload">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Proxima Nova</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border-radius: 9px;
	height: 22px;
	padding-left: auto;
	padding-right: auto;
}

QPushButton:hover {
	background-color: rgb(208, 208, 208);
	color: rgb(45, 45, 45);
}

QPushButton:pressed {
	background-color: rgb(207, 207, 207);
}</string>
          </property>
          <property name="text">
           <string>загрузить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: red;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
